Composer/packages/client/src/pages/language-generation/table-view.tsx

```diff
@@ -59,44 +65,47 @@ const TableView: React.FC<TableViewProps> = (props) => {
   );
 
   const onCreateNewTemplate = useCallback(() => {
-    const newName = increaseNameUtilNotExist(templates, 'TemplateName');
-    const payload = {
-      file,
-      projectId,
```

Comment:
Remove `projectId` here means we handle it in store layer, like use `current bot's projectId` ?

Reply:
Yes, we will use current bot's project id. And if composer want to switch to another bot, we flush all updates before opening a new bot.